Documentation/Signoff/LCD Block Subsystem.md

```diff
@@ -0,0 +1,104 @@
+# LCD Block Subsystem
+
+## Subsystem Function:
+
+The purpose of this subsystem is to help ease the user's understanding of the kit's functionalities, provide a digital user manual, provide the users with instructions or guided learning via challenges and lessons and enable a digital interaction with the car. Hence, this LCD Block Subsystem serves as the one of the primary interaction point between the user and the device as a whole. 
+
+## Constraints/Specifications:
+
+| Description       | Constraints/Specification:                                                                           | Source                         |
+|-------------------|------------------------------------------------------------------------------------------------------|--------------------------------|
+| Visibility        | The LCD must provide clear visibility in varying light conditions.                                   |  Interaction Design principles [3]             |
```

Comment:
How varying? How do we know that the chosen display can accomplish this?

Reply:
it also depends on the structure of LCD displays and power consumption. The brightness of ordinary TFT LCD screens is generally between 200-400,